pkg/client/client.go

```diff
@@ -238,6 +242,15 @@ func (c *Client) Fetch(ctx context.Context, request FetchRequest) error {
 				resp, err := stream.Recv()
 				if err == io.EOF {
 					c.Logger.Info("provider finished fetch", "provider", providerPlugin.Name(), "version", providerPlugin.Version())
+					// Print out partial fetch results if given
+					if len(partialFetchResults) != 0 {
```

Comment:
questioning if this is not better to be added in the FetchCallback? Because I would assume I would actually want to mark  the fetch as from V to ! in case I get partial fetch errors. Basically on each callback I can get an update of Errors rather than error.